Makefile

```diff
@@ -45,13 +45,21 @@ run:
 	poetry run python manage.py runserver
 
 docker-build:
-	docker build -t guide:latest --build-arg POETRY_INSTALL_ARGS="" -f Dockerfile .
+	docker-compose build
 
 docker-run:
-	docker run --name guide_latest -p ${PORT}:${PORT} --env-file .env guide:latest sh -c 'poetry run python manage.py runserver 0.0.0.0:${PORT}'
+	docker-compose up
 
-docker-exec:
-	docker exec -it guide_latest /bin/bash
+docker-shell:
+	docker-compose exec web bash
+
+docker-shell-frontend:
+	docker-compose exec frontend bash
 
 docker-init:
-	docker exec -it guide_latest /bin/bash -c "make backend"
+	docker-compose exec web poetry install
```

Comment:
Question: Why are dependencies being run like this? It happens during build time, which is probably fine.

```suggestion
```